File: CPP(C++)/Fibonacci.cpp
/* A program to form a fibonacci series */
#include <iostream>
using namespace std;

int main(int argc, char const *argv[])
{
	int nc, count=0, sum;
	int first, second;
	cout << "enter the no. of terms" << endl;
	cin >> nc;
	cout << "enter the first number" << endl;
	cin >> first;
	cout << "enter the second number"<<endl;
	cin >> second;
	while(count < nc){
		sum = first + second;
		cout << sum << endl;
		first = second;
		second = sum;
		count++;
	}
	return 0;
}

File: recursion/fibonacci.py
series = int(input("Enter no. of terms"))

cnt = 0
x, y = 0, 1

if series <= 0:
   print("Positive integer only")

elif series == 1:
   print(series)
   print(x)

else:
   print("Fibonacci series:")
   while cnt < series:
       print(x)
       nth = x + y
       x = y
       y = nth
       cnt += 1
